views/register.ejs

```diff
@@ -1,11 +1,27 @@
 <%- include("./partials/header") %>
 
-<h1>Sign up</h1>
-
-<form action="/register" method="POST">
-<input type="text" name="username" placeholder="username">
-<input type="password" name="password" placeholder="password">
-<button>Sign up</button>
-</form>
+<div class="container">
+    <div class="row">
+    <div class="col">
```

Comment:
Indentation is a bit off. Do not forget to format code before pushing it to the repo.